app/models/post_image.rb

```diff
@@ -5,6 +5,12 @@ class PostImage < ActiveRecord::Base
 
   mount_uploader :image, PostImageUploader
 
+  def image_url
```

Comment:
yes, but I tend to try to avoid `try` whenever possible. And if you search for where I used this method, it helps clean up the mess that was `hero && hero.image && hero.image.url.present?`